### PR TITLE
FEATURE NMH-91: Show All * versions if only exists [skip-test]

### PR DESCRIPTION
**Pull request by KshitizGIT**
Jira Ticket [NMH-91](https://johnsnowlabs.atlassian.net/browse/NMH-91)

### CODE REVIEW

File: docs/_frontend/src/models/FilterForm/FilterForm.js [hunk 1]
```diff
@@ -11,10 +11,36 @@ const FilterForm = ({ onSubmit, isLoading, meta, params }) => {
     ({ tasks, languages, editions } = meta.aggregations);
   }
   editions.sort((a, b) => {
-    if (a.indexOf('Healthcare') !== -1 && b.indexOf('Healthcare') === -1) {
-      return 1;
+    switch (a) {
```
Comment (pabla) [inline docs/_frontend/src/models/FilterForm/FilterForm.js:14]: Looks complicated. We could make it much simpler.

**Review by pabla — CHANGES_REQUESTED** [review]
Please apply prettier to the code.

File: docs/_frontend/src/models/FilterForm/FilterForm.js (hunk 1 above)
Comment (KshitizGIT) [reply docs/_frontend/src/models/FilterForm/FilterForm.js]: Great.. Thanks !

File: docs/_frontend/src/models/FilterForm/FilterForm.js [hunk 2]
```diff
@@ -3,19 +3,31 @@ import './FilterForm.css';
...
 
+const compareEditions = (a, b) => {
+  const getPriority = (edition) => {
+    const products = ['Spark NLP', 'Spark NLP for Healthcare', 'Spark OCR'];
+    const index = products.indexOf(edition);
+    if (index !== -1) {
+      return index - products.length;
+    }
+    if (edition.includes('Spark NLP for Healthcare')) {
+      return 0;
```
Comment (pabla) [inline docs/_frontend/src/models/FilterForm/FilterForm.js:14]: The order should be the same as for products:

1. Spark NLP versions
2. Spark NLP for Healthcare versions
3. Spark OCR versions

**Review by pabla — CHANGES_REQUESTED** [review]

File: docs/_frontend/src/models/FilterForm/FilterForm.js (hunk 2 above)
Comment (KshitizGIT) [reply docs/_frontend/src/models/FilterForm/FilterForm.js]: Ohh .. Done

**Review by pabla — APPROVED** [review]

**Review by maziyarpanahi — APPROVED** [review]

**Pull request merged**